internal/pkg/infrastructure/postgres/device.go

```diff
@@ -29,11 +29,6 @@ func (c *Client) AddDevice(d model.Device) (model.Device, errors.EdgeX) {
 		d.Id = uuid.New().String()
 	}
 
-	exists, _ := deviceNameExists(ctx, c.ConnPool, d.Name)
```

Comment:
keep these checks in the infrastructure layer

Reply:
Done